src/azul/terra.py

```diff
@@ -543,6 +558,9 @@ def _job_info(self, job: QueryJob, result: RowIterator) -> JSON:
             'query': self._trunc_query(job.query)
         }
 
+    def _job_has_result(self, job_info: JSON) -> bool:
+        return 'totalBytesProcessed' in json_mapping(job_info['stats'])
```

Comment:
A comment needs to explain our thinking here as it constitutes empirical use of undocumented behavior.